packages/platform-express/adapters/express-adapter.ts

```diff
@@ -78,7 +80,17 @@ export class ExpressAdapter extends AbstractHttpAdapter {
       ) {
         response.setHeader('Content-Length', streamHeaders.length);
       }
-      return body.getStream().pipe(response);
+      return pipeline(
+        body.getStream().on('error', (err: Error) => {
```

Comment:
```suggestion
        body.getStream().once('error', (err: Error) => {
```

to prevent the error 'Cannot set headers after they are sent to the client' if for whatever reason the _error_ event is emitted multiple times (not sure if this is possible tho)